packages/core/components/DraggableComponent/index.tsx

```diff
@@ -434,12 +434,21 @@ export const DraggableComponent = ({
 
           const diffLeft = rect.x;
           const exceedsBoundsLeft = diffLeft < 0;
+          const diffTop = rect.y;
+          const exceedsBoundsTop = diffTop < 0;
 
           // Modify position if it spills over frame
           if (exceedsBoundsLeft) {
             el.style.transformOrigin = "left top";
             el.style.left = "0px";
           }
+
+          if (exceedsBoundsTop) {
+            el.style.top = "0px";
```

Comment:
Could you try setting this as `12px` so the action label doesn't butt-up against the component?

I also thought about moving it up to the top of the component, but then there's a risk the overlay awkwardly covers the contents, especially with smaller items.

Without 12px:

<img width="863" alt="image" src="https://github.com/user-attachments/assets/99d6aafa-5876-4e2d-b324-0dd4815b5341" />

With 12px:

<img width="857" alt="image" src="https://github.com/user-attachments/assets/ded7484b-9822-40ee-bb15-9cdcc32c8f56" />